db_assessment/optimusprime.py

```diff
@@ -309,13 +332,35 @@ def argumentsParser():
 
     parser.add_argument("-importcomment", type=str, default='', help="Comment for the Import")
 
+
+    # SM: 04/28/2022:[Bug#90] Recreate views without loading the data:
+    parser.add_argument("-recreateviews", default=False, help="Recreate views without loading the data (Yes/No)")
```

Comment:
Should it be True or False?

Reply:
Instead of Yes/No because of 
==
default=False
==

Reply:
@sandeepmanocha @ersantosnet when I run it fails with below error:

```
(op-venv) ~/Desktop/OP/Issues/addflagsskipdataload/oracle-database-assessment/db_assessment$ python3 optimusprime.py -dataset bt_pull147 -fileslocation /Users/pawarsachin/Desktop/OP/Issues/beautifultb/op190_203   -projectname oraclepssupport -recreateviews Yes -dbversion 190 -collectionid ""
Traceback (most recent call last):
  File "optimusprime.py", line 391, in <module>
    runMain(args)
  File "optimusprime.py", line 82, in runMain
    if import_db_assessment.checkDataSetExists(bqDataset, gcpProjectName):
AttributeError: module 'import_db_assessment' has no attribute 'checkDataSetExists'
(op-venv) ~/Desktop/OP/Issues/addflagsskipdataload/oracle-database-assessment/db_assessment$ cd ..
(op-venv) ~/Desktop/OP/Issues/addflagsskipdataload/oracle-database-assessment$ grep checkDataSetExists */*
grep: db_assessment/__pycache__: Is a directory
grep: db_assessment/dbSQLCollector: Is a directory
grep: db_assessment/opConfig: Is a directory
db_assessment/optimusprime.py:        if import_db_assessment.checkDataSetExists(bqDataset, gcpProjectName):
```

Is there any code for 'checkDataSetExists' that got missed to be added to the commit?